local/compose/create.go

```diff
@@ -335,6 +368,15 @@ func getVolumesFrom(project *types.Project, volumesFrom []string) ([]string, []s
 
 }
 
+func getNetworkFrom(networkFrom string) string {
```

Comment:
unclear naming. parameter is actualy a network mode, `networkFrom` => `networkMode` 
method name is also unclear, doesn't return a network but dependent service
=> `getDependentServiceByNetwork` or any (maybe shorter :P) name

Reply:
Done